dockstore-webservice/src/main/java/io/dockstore/webservice/core/Workflow.java

```diff
@@ -122,6 +123,11 @@
     @ApiModelProperty(value = "This is a descriptor type for the workflow, by default either CWL, WDL, NFL, or gxformat2 (Defaults to CWL).", required = true, position = 18, allowableValues = "CWL, WDL, NFL, gxformat2, service")
     private DescriptorLanguage descriptorType;
 
+    @Column
+    @Convert(converter = DescriptorLanguageSubclassConverter.class)
+    @ApiModelProperty(value = "This is a descriptor type subclass for the workflow. Currently it is only used for services.", position = 19)
+    private DescriptorLanguageSubclass descriptorTypeSubclass;
```

Comment:
I'd prefer this to be non-nullable and just set to a `n/a` term or similar by default.

Reply:
Would I have a subclass called not applicable?